kerosene/nn/criterions.py

```diff
@@ -218,6 +222,139 @@ def ignore_index_fn(dice_vector):
         return dice
 
 
+class TverskyLoss(_Loss):
```

Comment:
Are the twersky loss tested ? Where is the implementation coming from ?

Reply:
Implementation is exactly the same as the Dice Loss, but with two coefficients more (alpha and beta). 
But no, not tested. Will write a test with alpha and beta = 0.5 which corresponds to the Dice Loss. Otherwise, would be a bit hard to test.

Reply:
Test done.